rust/connlib/libs/tunnel/src/control_protocol.rs

```diff
@@ -145,12 +146,11 @@ where
         gateway_id: Id,
         resource_id: Id,
     ) {
-        tracing::trace!("Peer Connection State has changed: {state}");
+        tracing::trace!(?state, "peer_state");
```

Comment:
The state should already be in the `span` right? Seems redundant to log it here too.